### PR TITLE
test: New libreport uses bugzilla API key

### PR DESCRIPTION
**Pull request by marusak**

### CODE REVIEW

File: test/verify/check-system-journal [hunk 1]
```diff
@@ -643,8 +644,15 @@ ExecStart=/bin/sh -c 'sleep 5; for s in $(seq 10); do echo SLOW; sleep 0.1; done
         b.click("#abrt-reporting .pf-l-split:contains('Report to Cockpit') button:contains('Report')")
 
         test_user = 'correcthorsebatterystaple'
+        fields = ['text', 'password']
 
-        for purpose in ['text', 'password']:
+        # Since libvirt version `2.15.2-7` bugzilla plugin does not require username and password but API key
```
Comment (martinpitt) [inline test/verify/check-system-journal]: It has nothing to do with libvirt, right? (comment/commit message)

File: test/verify/check-system-journal [hunk 2]
```diff
@@ -643,8 +644,15 @@ ExecStart=/bin/sh -c 'sleep 5; for s in $(seq 10); do echo SLOW; sleep 0.1; done
         b.click("#abrt-reporting .pf-l-split:contains('Report to Cockpit') button:contains('Report')")
 
         test_user = 'correcthorsebatterystaple'
+        fields = ['text', 'password']
 
-        for purpose in ['text', 'password']:
+        # Since libreport version `2.15.2-7` bugzilla plugin does not require username and password but API key
+        # See https://github.com/abrt/libreport/commit/7c3c5d1ae1c39f0a3a3e9730bd25643a3f04fd50
+        libreport_version = m.execute('rpm -qa --qf "%{VERSION}-%{RELEASE}" libreport-plugin-bugzilla').rsplit(".", 1)[0]
+        if version.parse(libreport_version) > version.parse("2.15.2-6"):
```
Comment (martinpitt) [inline test/verify/check-system-journal:652]: Could this be the new default, and we handle user+password in fedora-35 (and possibly fedora-36, didn't check) as the "special case"? Then it's easier to spot with grepping the tests for these OS names, and we can clean it up over time.

Comment (martinpitt) [reply test/verify/check-system-journal:652]: I made a calendar entry in a month to clean this up.

**Review by martinpitt — APPROVED** [review]
Dakujem!

**Comment by martinpitt** [comment]
Huh @ [c9s failure](https://artifacts.dev.testing-farm.io/927d997f-4d14-4c4e-8f37-6abe7f1ab7f3/):
```
  File "test/verify/check-system-journal", line 23, in <module>
    from packaging import version
ModuleNotFoundError: No module named 'packaging'
```
Last-minute Python regression? Unrelated to this PR, but needs to be investigated.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Oh wait, of course the packit failure *is* related -- it needs to add the new python3-packaging test dependency. Not sure if it is present on our images already. If not, I'd rather suggest to avoid it completely, and parse it ourselves. This hack won't have to stay for long.

**Comment by marusak** [comment]
Yes, it is related. We need to have it in our task container - which we have. I was not thinking of testing farm. Let me add deps.

Also `TestJournal.testBasic` failed on [rhel-9-0](https://logs.cockpit-project.org/logs/pull-17049-20220225-081848-73397fa8-rhel-9-0/log.html), [debian-testing](https://logs-https-frontdoor.apps.ocp.ci.centos.org/logs/pull-17049-20220225-081848-73397fa8-debian-testing/log.html), [arch](https://logs.cockpit-project.org/logs/pull-17049-20220225-081848-73397fa8-arch/log.html) and [ubuntu-2004](https://logs.cockpit-project.org/logs/pull-17049-20220225-081848-73397fa8-ubuntu-2004/log.html). Always the same way. That cannot be coincidence, there must be more to it. Looking.

**Comment by marusak** [comment]
waait, that? it expects to see in journal `Started slow10.service - Slowly log 10 identical lines.` but it sees `Started Slowly log 10 identical lines.`
How is this related to this change? It seems like image refresh issue - but how did that pass tests??

**Comment by martinpitt** [comment]
[this failure](https://logs.cockpit-project.org/logs/pull-17049-20220228-112310-5d284728-fedora-35-devel/log.html#297-2) is in journal tests, but almost certainly unrelated to the change here. The ten thousand key collisions are also not new. Retrying.

**Review by martinpitt — APPROVED** [review]
Cheers!

**Pull request merged**